channeldb/channel.go

```diff
@@ -293,6 +295,12 @@ func (c *OpenChannel) UpdateCommitment(newCommitment *wire.MsgTx,
 		if err := putChanCapacity(chanBucket, c); err != nil {
 			return err
 		}
+		if err := putChanSatoshisSent(chanBucket, c); err != nil {
```

Comment:
As pointed out in my comment below, rather than calling these functions here, `putChanTotalFlow` should instead be called with the `chanBucket` as its first argument.